docs/src/pages/guides/csp.md

```diff
@@ -1,25 +1,52 @@
-# Content Security Policy
+# Content Security Policy (CSP)
 
-Starting in JSS version 9.6.0, Material-UI supports Content Security Policy headers.
+Starting with JSS version 9.6.0, Material-UI supports Content Security Policy headers.
 
 ## What is CSP and why is it useful?
 
-Basically, CSP mitigates cross-site scripting (XSS) attacks by requiring developers to whitelist the sources their assets are retrieved from. This list is returned as a header from the server. For instance, say you have a site hosted at `https://website.example` the CSP header `default-src: 'self';` will allow all assets that are located at `https://website.example/*` and deny all others. If there is a section of your website that is vulnerable to XSS where unescaped user input is displayed, an attacker could input:
+Basically, CSP mitigates cross-site scripting (XSS) attacks by requiring developers to whitelist the sources their assets are retrieved from. This list is returned as a header from the server. For instance, say you have a site hosted at `https://website.example` the CSP header `default-src: 'self';` will allow all assets that are located at `https://website.example/*` and deny all others. If there is a section of your website that is vulnerable to XSS where unescaped user input is displayed, an attacker could input something like:
```

Comment:
Hard to see what changed as it's one lone unwrapped line, but the change I requested was to use the canonical "[https://example.com/](https://example.com/)"